Web/Controllers/ApiController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Threading.Tasks;
+using ICSharpCode.CodeConverter.Util;
+using ICSharpCode.CodeConverter.Web.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ICSharpCode.CodeConverter.Web.Controllers
+{
+    [Route("api")]
+    public class ApiController : Controller
+    {
+        [HttpPost]
+        [Route("Convert")]
+        [Route("Converter")] //Old name, kept for compatibility
```

Comment:
I wasn't sure if anyone uses this existing API endpoint within the app so have left it here for now.